packages/mobile/src/tokens/saga.ts

```diff
@@ -328,6 +328,17 @@ export function* calculateTotalTokenBalance() {
   yield put(setTotalTokenBalance(totalBalance.toFixed(2).toString()))
 }
 
+export function* tokenAmountInWei(amount: BigNumber, tokenAddress: string) {
```

Comment:
Shouldn't this be called `tokenAmountInSmallestUnit` or something?
`1 wei` is really only `10^-18 eth`, right?